packages/safe-core-sdk/src/managers/ownerManager.ts

```diff
@@ -53,7 +53,8 @@ class OwnerManager {
   }
 
   async getOwners(): Promise<string[]> {
-    return this.#safeContract.getOwners()
+    const owners = await this.#safeContract.getOwners()
+    return [...owners]
```

Comment:
why does it have to be this way?

Reply:
Otherwise it returns a frozen array.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze
`"As an object, an array can be frozen; after doing so, its elements cannot be altered and no elements can be added to or removed from the array."`

Reply:
but it doesn't matter in our case, does it?

Reply:
Not in our case, but I think it will avoid some confusion for many developers if they try to sort the array or modify it and it does not work.